src/data/nav-items.yaml

```diff
@@ -201,7 +201,7 @@
   pages:
     - title: Overview
       path: /community/overview
-    - title: Component index
+    - title: Component Index
```

Comment:
Are we sure about this? "Data visualization", "Bugs and requests", "Add ons", "Migration guide" etc. all make me think it's find being lower case `index`

Reply:
just updated lol it was an accident